20/streams/developer-guide/dsl-api.html

```diff
@@ -3290,6 +3290,32 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
             <pre class="brush: scala;">
               libraryDependencies += "org.apache.kafka" %% "kafka-streams-scala" % "{{fullDotVersion}}"
             </pre>
+            <p>
+              <strong>Notes</strong>:
+              <ul>
+                <li>
+                  The bugfix version <code>2.0.1</code> fixed several important flaws in  <code class="docutils literal"><span class="pre">kafka-streams-scala</span></code>.
+                  It's strongly recommended to use the latest bugfix release in general and to avoid <code>2.0.0</code> specifically
```

Comment:
nit: `avoid 2.0.0` -> `avoid the 2.0.0 version of  kafka-streams-scala`

Reply:
Thanks. I'll make this change in all the PRs after I finish collecting reviews.